pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md

```diff
@@ -18,7 +18,7 @@ After adopting the cluster, you use this AMI to scale up, or replace a failed in
 
 <p class="message--note"><strong>NOTE: </strong>Konvoy v1.8.4 and v1.8.5 uses Kubernetes version 1.20.13 by default. If upgrading from Konvoy v1.8.3, use Kubernetes version 1.20.11 for the below commands.</p>
 
-1.  Create the AMI using [Konvoy Image Builder][kib] (you must [use the Konvoy Image Builder v1.5.0 release, downloadable from GitHub][kib-releases]):
+1.  Create the AMI using [Konvoy Image Builder][kib] (you **must [use the Konvoy Image Builder v1.5.0 release, downloadable from GitHub][kib-releases]**):
```

Comment:
I'm not a fan of this @msdolbey we can't just add version numbers like this throughout the docs. We need a better strategy for KIB versioning.